.travis.yml

```diff
@@ -1,15 +0,0 @@
-language: node_js
-
-node_js: stable
-
-script:
-  - npm run test-browsers
-
-addons:
-  hosts:
-    - airtap.local
```

Comment:
not sure what this does

Reply:
It adds the `airtap.local` hostname to `/etc/hosts`. To do the same in GitHub Actions and also add the required secrets, you can copy https://github.com/nodejs/readable-stream/blob/bcbe5e06b10a8915f622e6530f057f7a4ee1a09e/.github/workflows/sauce.yml#L13-L24